src/pudl/output/ferc1.py

```diff
@@ -2288,10 +2288,12 @@ def propagate_node_attributes(self: Self, annotated_forest: nx.DiGraph):
 
         Propagate tags leafwards, rootward &  to the _correction nodes.
         """
+        tags_to_propagate = ["in_rate_base", "rate_base_category"]
         ## Leafwards propagation
-        annotated_forest = _propagate_tags_leafward(annotated_forest, ["in_rate_base"])
+        annotated_forest = _propagate_tags_leafward(annotated_forest, tags_to_propagate)
         # Rootward propagation
-        annotated_forest = _propagate_tag_rootward(annotated_forest, "in_rate_base")
+        for tag in tags_to_propagate:
+            annotated_forest = _propagate_tag_rootward(annotated_forest, tag)
```

Comment:
this adds the `rate_base_category` into the set of tags that we are propagating up and down the forest.